css/main.css

```diff
@@ -1,3 +1,96 @@
-/* Add your custom CSS here */
-
-/* Dont' forget to link this file to your HTML in the <head> */
+body {
+    color: #2F2F2E
+}
+.buttons {
+    background-color: #CFF4EC;
+    border: #CFF4EC;
+    padding: 10px 30px 10px 30px;
+    margin: 30px 0 20px 10px;
+    font-family: 'Quicksand', sans-serif;
+}
+.buttons:hover {
+    background-color: #4D7369;
+    color: white;
+}
+#booking{
+    float: right
+}
+h1, h2 {
+    font-family: 'Libre Baskerville', serif;
+    padding: 20px 0 10px 0;
+}
+h3 {
+    font-family: 'Libre Baskerville', serif;
+    font-style: italic;
+    padding-top: 10px;
+}
+p {
+    font-family: 'Quicksand', sans-serif;
+}
+a {
+    text-decoration: none;
+    color: #2F2F2E
+}
+.nav-link:hover {
+    color: #94D6C8;
+  }
+.introduction {
+    background-color: #CFF4EC;
+    padding: 25% 0 25%;
+}
+.couples {
+    background-color: #CFF4EC;
+    padding: 30px;
+}
+.about-maria, .articles {
+    background-color: #F2F2F1;
+    padding: 30px;
+}
+.individual {
+    background-color: #E2F6CD;
+    padding: 30px;
+}
+.group {
+    background-color: #F2F2F1;
+    padding: 30px;
+}
```

Comment:
Remember to make CSS classes as re-usable as possible. This means you could have one class for the styles these blocks have in common, and separate classes for what differentiates them. 
Example:
```
.service-block { 
   padding: 30px;
}
.light-blue { 
   background-color: #CFF4EC;
} 
.light-grey { 
   background-color: #F2F2F1;
} 
.light-green { 
   background-color: #E2F6CD;
} 
```
and then you can set them in your HTML like this:
```
  <section class="service-block light-blue">
    <!-- blue content -->
  </section>

  <section class="service-block light-grey">
    <!-- grey content -->
  </section>

  <section class="service-block light-green">
    <!-- green content -->
  </section>
```